docs/source/changelog.rst

```diff
@@ -1,3 +1,9 @@
+1.0.5
```

Comment:
Did you mean this to be ``1.0.6``?

Reply:
Oh whoops, I definitely did. Just wanted to get a start on adding entries.

I do need some education on how the downstream portion works, but we can talk about that elsewhere. :)